chart_schema.yaml

```diff
@@ -0,0 +1,38 @@
+# yamllint disable rule:document-start
+name: "str()"
```

Comment:
Hm, when I download the file, these lines are also modified to add quotes. Not the end of the world, but I think if we don't actually need to modify this file, maybe we should download it as part of the toolchain and then copy it here? We can also add it to gitignore and don't add it to the set of YAML files we lint.

Reply:
sure